lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -280,11 +341,101 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         this.categoryControlVisible = false;
     }
 
-    cancelChanges() {
+    cancelChanges(buttonType: ButtonType, event: MouseEvent) {
+        event.stopPropagation();
+        this.toggleEditMode(buttonType);
         this.revertChanges();
         this.loadProperties(this.node);
     }
 
+    isEditingPanel(): boolean {
+        return (
+            (this.editable && this.hasMetadataChanged) ||
+            (this.editableTags && this.hasMetadataChanged) ||
+            (this.editableCategories && this.hasMetadataChanged) ||
+            (this.editableGroup?.editable && this.hasMetadataChanged)
+        );
+    }
+
+    showSnackbar(message: string): void {
+        this.notificationService.showError(message);
+    }
+
+    toggleEdit(event: MouseEvent, group: CardViewGroup, buttonType: ButtonType): void {
+        if (this.isEditingPanel()) {
+            this.showSnackbar('METADATA.BASIC.SNACKBAR_MESSAGE');
```

Comment:
are you raising an error here? the "showSnackbar" does not give a clue, and translation key too